tests/functional/cypress/e2e/about.cy.js

```diff
@@ -4,7 +4,8 @@ const baseUrl = Cypress.env('baseUrl');
 
 describe('Application About Page', () => {
   it('Visits the app about page', () => {
-    cy.visit(`/${depEnv}`);
+    //cy.visit(`/${depEnv}`);
+    cy.visit('https://chefs-dev.apps.silver.devops.gov.bc.ca/app/');
```

Comment:
If this is deployed to the PR environment, it will always fail since the /app wouldn't exist. It would be /pr-X

Reply:
will fix